code/game/turfs/trench.dm

```diff
@@ -0,0 +1,31 @@
+obj/structure/trench
+	name = "Trench"
+	desc = "A wide, narrow earthwork designed to protect the user from any projectiles"
+	icon = 'icons/turf/trench.dmi'
+	icon_state = "gold0"
+	density = FALSE
+	anchored = TRUE
+	var/slowamt = 4
+	tiles_with = list(
+		/obj/structure/trench,
+	)
+
+turf/open/ground/trenchturf
+	name = "Trench Floor"
+	desc = "Thats a Trench Floor"
```

Comment:
What is the trench floor specifically, and what's the difference between trenches?

Reply:
It must be a remnant from way back during early development, I will get rid of it.